static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -478,12 +478,11 @@ $block-padding-right: $gs-gutter;
 .dropdown--live-feed {
     border-top: 0;
 
+    $content-gutter: $gs-gutter/2;
+
     @include mq($until: desktop) {
-        margin-right: $gs-gutter*-1;
-        margin-left: $gs-gutter*-1;
-        padding-right: $gs-gutter;
-        padding-left: $gs-gutter;
-        padding-top: $gs-baseline*4;
+        margin-right: ($content-gutter)*-1;
```

Comment:
it's not certain these two things will always be laid out with the same relationship, though, is it? 

by tying them both to a new variable, changing one necessarily changes the other - isn't that the kind of side-effect that we want to remove from our current scss?